tests/unit/literaturesuggest/test_literaturesuggest_tasks.py

```diff
@@ -0,0 +1,693 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+import mock
+import pytest
+
+from inspirehep.modules.literaturesuggest.tasks import (
+    formdata_to_model,
+    new_ticket_context,
+    reply_ticket_context,
+    curation_ticket_context,
+    curation_ticket_needed,
+)
+
+
+class StubObj(object):
+    def __init__(self, data, extra_data, id=1, id_user=1):
+        self.data = data
+        self.extra_data = extra_data
+        self.id = id
+        self.id_user = id_user
+
+
+class DummyEng(object):
+    pass
+
+
+class StubUser(object):
+    def __init__(self, email):
+        self.email = email
+
+
+@mock.patch('inspirehep.modules.literaturesuggest.tasks.User')
+@mock.patch('inspirehep.modules.literaturesuggest.tasks.UserIdentity')
+def test_formdata_to_model_populates_schema(u, ui):
+    data = {}
+    extra_data = {}
+    formdata = {
+        'type_of_doc': 'foo',
+        'title': [
+            'bar',
+        ],
+    }
+
+    obj = StubObj(data, extra_data)
+
+    expected = 'http://localhost:5000/schemas/records/hep.json'
+    result = formdata_to_model(obj, formdata)
+
+    assert expected == result['$schema']
```

Comment:
They do produce something in `result`, but I'm not testing for that. They have to be there, otherwise the method under test throws a `KeyError` or something like that.